### DISCUSSION TITLE
Update Trim Like Open Tx

### DISCUSSION THREAD

**Discussion by Sahartman21** [FrSkyRC/ETHOS-Feedback-Community#3114] [Ideas]
Thank you for supporting upgrades in ethos. I fly models where flight modes 0, 1, and 2 are used within a couple of minutes. It is desirable to have like used for open TX. 

Use own trim
Use FM0 trim
Use FM0 plus own trim as offset.

![received_910215726640407](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/152731487/5d8f5254-bd3c-4530-a605-d0f3791c277c)
![received_1727779121067103](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/152731487/513adf02-39d9-47e2-bfe2-6efbde094b37)
![received_872939607666477](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/152731487/57c5280d-90cb-4cc6-a356-4e7b4d3f0d6a)

**Comment by bsongis-frsky** [comment]
One of the 1.5.0 features!
